.external

```diff
@@ -1,2 +1,2 @@
-xamarin/monodroid:master@ad19471b8478f9f8028c9a3517ad3ca7bca5fb4a
+xamarin/monodroid:master@390c048caaddfc6549ef94d6a80f71e5aa9416bc
```

Comment:
can you change this to this commit d9fcaa019c5e2fb14384d47040bbe7f0b8e7695c?  its the head of master. It should also fix a few failing tests.

Reply:
Done